bcda/models/models.go

```diff
@@ -530,6 +536,55 @@ func (cclfBeneficiary *CCLFBeneficiary) GetBlueButtonID(bb client.APIClient) (bl
 	return blueButtonID, nil
 }
 
+// This method will ensure that a valid BlueButton ID is returned.
+// If you use suppressionBeneficiary.BlueButtonID you will not be guaranteed a valid value
+func (suppressionBeneficiary *Suppression) GetBlueButtonID(bb client.APIClient) (blueButtonID string, err error) {
```

Comment:
Should refactor this so that both flavors of `GetBlueButtonID()` use the same code... i.e., the one called on the `cclfbeneficiary` model and the one called on the `suppressionbeneficiary` model.

Reply:
agreed! Like a common base function called by 2 different methods. Nice